test/e2e-test/suite_test.go

```diff
@@ -170,6 +170,20 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(k8sClient.Create(context.Background(), &wd)).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
 	By("Created containerizedworkload.core.oam.dev")
 
+	// create workload definition for 'deployments'
+	wdDeploy := v1alpha2.WorkloadDefinition{
```

Comment:
With these webhooks enabled, it will always need WorkloadDefintion during creating AppConfig.